test/raw_explain/test_raw_explanation.py

```diff
@@ -30,6 +30,10 @@ def test_get_global_raw_explanations_classification(self, iris, tabular_explaine
         global_raw_explanation = global_explanation.get_raw_explanation(
             [feature_map], raw_feature_names=feature_names[:feature_map.shape[0]])
 
+        assert global_raw_explanation.expected_values == global_explanation.expected_values
+        assert global_raw_explanation.init_data == global_explanation.init_data
```

Comment:
This points out something that I hadn't given enough thought earlier - right now, the return type for `init_data` and `eval_data` is `str or list`, but I'm not sure that's how it should be. Probably it should be in the original form. What do you think?

Reply:
good point, it should probably be the original form - maybe not str (but why is it list right now, do we convert pandas/numpy to list?)

Reply:
It's confusing because it could be a string if they had previously uploaded a Dataset. I guess it's in a list because we jsonify it. Maybe this will be fixed if we remove the Datasets mixin from this repo?

Reply:
I just updated the test to do an np.all since the init_data is a numpy array in this case